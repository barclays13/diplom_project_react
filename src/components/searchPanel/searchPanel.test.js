import React from 'react';
import SearchPanel from './searchPanel';
import {shallow} from 'enzyme';


describe('Testing <SearchPanel>', () => {
    it ('SearchPanel have renderer correctly', () => {
    const search = shallow(<SearchPanel/>);
    expect(search).toMatchSnapshot();
    })
});