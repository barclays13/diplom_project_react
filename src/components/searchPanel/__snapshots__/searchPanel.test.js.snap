// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <SearchPanel> SearchPanel have renderer correctly 1`] = `
<form
  action="#"
  className="shop__search"
>
  <label
    className="shop__search-label"
    htmlFor="filter"
  >
    Looking for
  </label>
  <input
    className="shop__search-input"
    id="filter"
    onChange={[Function]}
    placeholder="start typing here..."
    type="text"
  />
</form>
`;
